grade.go

```diff
@@ -392,39 +404,27 @@ func main() {
 
 		result.StudentId = hydratedStudent.Id
 		result.AssignmentId = int8(1)
-
-		if language == "python3" || language == "python" {
+		
+		if language == "python3" || language == "python" || language == "javascript" || language == "js" {
 			result.CompileSuccess = true
-
-			stdout := runInterpreted(filepath.Join(workDir, dir), runArgs, input)
+	
+			stdout := runInterpreted(filepath.Join(workDir, dir), runArgs, language, input)
 			fmt.Printf("Output for %s: %s", result.Student, stdout)
 			result.RunCorrect, result.Feedback = compare(expected, stdout)
```

Comment:
This should be using `processOutput()` instead of `compare()`, unless for some reason processOutput isn't working with the interpreted languages. Not gonna block since it's an easy fix, but make sure to test it before the fix is pushed.
```go
result.RunCorrect, result.Feedback = processOutput(expected, stdout)
```

Reply:
Will definitely review this in the next session!